README.md

```diff
@@ -171,38 +171,36 @@ API endpoint for user authentication.
 
 ```json
 {
-  "user": {
-    "id": "97",
-    "phone": "555-555-5555",
-    "email": "example@test.com",
-    "userTypeId": "3",
-    "name": "John Doe",
-    "firstName": "John",
-    "lastName": "Doe",
-    "dateOfBirth": "01011983",
-    "sex": "male",
-    "verifiedEmail": "0",
-    "verifiedPhone": "0",
-    "active": "1",
-    "archived": "0",
-    "created": "2018-10-01 03:24:43",
-    "updated": "2018-10-01 03:24:43",
-    "deleted": "0"
-  },
-  "session": {
-    "accessToken": "07ywyJ6TdsMLOSh2GTfq328VyYEg6C3L",
-    "expirationMinutes": 300,
-    "UTCExpirationTime": "2019-10-16 18:52:27",
-    "PTExpirationTime": "2019-10-16 18:52:27"
-  }
+    "status": {
+        "status_code": 200,
+        "message": "SUCCESS"
+    },
+    "data": {
+        "user": {
+            "id": 4,
+            "email": "jayanta.5056@gmail.com",
+            "authorities": [
+                {
+                    "authority": "ROLE_USER"
+                }
+            ],
+            "username": "Jayanta"
+        },
+        "session": {
+            "accessToken": "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJKYXlhbnRhIiwiaWF0IjoxNTk2NjE0OTI5LCJleHAiOjE1OTY3MDEzMjl9.doC6QumEu_c_iF3kgd9Cb7vHyOYw3bFn2gTNVSLJP_M76DOyB7-Ay4KFgIIX4Cd6ZwXtt1ovYOLxjqQsQz72qw",
+            "expirationMiliSecond": 86400000,
+            "utcExpirationTime": "2020-08-06T08:08:49.119+00:00",
+            "ptExpirationTime": "2020-08-06T08:08:49.119+00:00"
+        }
+    }
 }
 ```
 </details>
 
 [table of contents](#table-of-contents)
 
 ### Sign Out 
-Endpoint: `/auth/signout`
+Endpoint: `/api/auth/logout`
```

Comment:
@Jayanta-Redapple It would be nice if we maintained the business language of `signin`, `signup`, `signout`. Refactoring the logout function to signout would be good for this consistency.